config/backends.go

```diff
@@ -165,8 +167,13 @@ type RedisTLS struct {
 }
 
 func (cfg *Redis) validateAndLog() error {
-	log.Infof("config.backend.redis.host: %s", cfg.Host)
-	log.Infof("config.backend.redis.port: %d", cfg.Port)
+	if cfg.Host != "" && len(cfg.SentinelHostsPorts) > 0 {
+		log.Infof("config.backend.redis.sentinel_hosts_ports: %s. Note that redis 'host' will be ignore if 'sentinel_hosts_ports' is define", cfg.SentinelHostsPorts)
+		log.Infof("config.backend.redis.sentinel_mastername: %s.", cfg.SentinelMasterName)
```

Comment:
@cyrinux thank you for your addresing our feedback. When we want to configure Redis in Sentinel mode, is `SentinelMasterName` a required field? In other words, if `len(cfg.SentinelHostsPorts) > 0` is true, do we also need `cfg.SentinelMasterName` to not be empty?

If we do, then I believe an extra `cfg.SentinelMasterName != ""` check should be added in case `len(cfg.SentinelHostsPorts) > 0` is found to be true and `log.Infof()` or error out accordingly.

Thoughts?